src/features/authentication/public/LoginView.tsx

```diff
@@ -0,0 +1,117 @@
+import { Alert, Box, Container, Grid, Link, TextField } from '@mui/material';
+import { animated, config, useTransition } from 'react-spring';
+import styled, { createGlobalStyle } from 'styled-components';
+
+import SubmitButton from '~/core/components/SubmitButton';
+import ToastBar from '~/core/components/ToastBar';
+import FadeElementInDown from '~/core/components/animation/FadeInDown';
+import useLoginForm from '~/features/authentication/public/hooks/useLoginForm';
+
+const RootCss = createGlobalStyle`
+    html, body, #root {
+        width: 100%;
+        height: 100%;
+        margin: 0;
+        padding: 0;
+    }
+`;
+
+const Root = styled(Container)`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 100%;
+`;
+
+const AnimatedErrorBox = animated(styled(Box)`
+    margin-top: -3rem;
+`);
+
+const GridLeft = styled(Grid)`
+    text-align: left;
+`;
+
+const FormBox = styled(Box)`
+    display: flex;
+    flex-direction: column;
+    margin-top: 3rem;
+    width: 20rem;
+`;
+
+function LoginView() {
```

Comment:
As you can see below, this login page view was greatly pared down from the PKA version. I was able to clean some stuff up (like by using the `useTransition()` below), and also just made it more generic/basic, so that it can easily be adapted for the needs of app, since our Login page designs often vary quite a bit in look and feel from app to app.

I also extracted the majority of the logic into a separate `useLoginForm()` hook, to further help the separation of it from the presentation, as well as to make it easy to have multiple login points but also keep things DRY (for example if an app also had some sort of mini login form in a pop-over or something in addition to a /login route.)